test/uk/gov/hmrc/driverinspectionnotificationfrontend/controllers/SearchResultControllerSpec.scala

```diff
@@ -259,20 +261,20 @@ class SearchResultControllerSpec extends ControllerBaseSpec {
           )
             .thenReturn(
               List(
-                Right(InspectionType("1", "CUSTOMS")      -> List(Right(locationForCustoms))),
-                Right(InspectionType("2", "DEFRA")        -> List(Right(location))),
-                Right(InspectionType("3", "TRANSIT")      -> List(Right(location))),
-                Right(InspectionType("4", "OGD")          -> List(Right(location))),
-                Right(InspectionType("5", "DEFRA_PLANTS") -> List(Right(location))),
-                Right(InspectionType("6", "ATA")          -> List(Right(location))),
-                Right(InspectionType("7", "SAD")          -> List(Right(location))),
-                Right(InspectionType("8", "TIR")          -> List(Right(location))),
-                Right(InspectionType("9", "DBC")          -> List(Right(locationForCustoms))),
-                Right(InspectionType("10", "EIDR")        -> List(Right(locationForCustoms))),
-                Right(InspectionType("11", "EXEMPTION")   -> List(Right(locationForCustoms))),
-                Right(InspectionType("12", "EMPTY")       -> List(Right(locationForCustoms))),
-                Right(InspectionType("13", "DAERA")       -> List(Right(location))),
-                Right(InspectionType("16", "SnS")         -> List(Right(location)))
+                Right(InspectionTypeWithLocations(InspectionType("1", "CUSTOMS"), List(Right(locationForCustoms)))),
+                Right(InspectionTypeWithLocations(InspectionType("2", "DEFRA"), List(Right(location)))),
+                Right(InspectionTypeWithLocations(InspectionType("3", "TRANSIT"), List(Right(location)))),
+                Right(InspectionTypeWithLocations(InspectionType("4", "OGD"), List(Right(location)))),
+                Right(InspectionTypeWithLocations(InspectionType("5", "DEFRA_PLANTS"), List(Right(location)))),
+                Right(InspectionTypeWithLocations(InspectionType("6", "ATA"), List(Right(location)))),
+                Right(InspectionTypeWithLocations(InspectionType("7", "SAD"), List(Right(location)))),
+                Right(InspectionTypeWithLocations(InspectionType("8", "TIR"), List(Right(location)))),
+                Right(InspectionTypeWithLocations(InspectionType("9", "DBC"), List(Right(locationForCustoms)))),
+                Right(InspectionTypeWithLocations(InspectionType("10", "EIDR"), List(Right(locationForCustoms)))),
+                Right(InspectionTypeWithLocations(InspectionType("11", "EXEMPTION"), List(Right(locationForCustoms)))),
+                Right(InspectionTypeWithLocations(InspectionType("12", "EMPTY"), List(Right(locationForCustoms)))),
+                Right(InspectionTypeWithLocations(InspectionType("13", "DAERA"), List(Right(location)))),
+                Right(InspectionTypeWithLocations(InspectionType("16", "SnS"), List(Right(location))))
```

Comment:
not sure what this test for but if the intention is to check all inspectionTypes then worth adding `18` too?

Reply:
the test is checking for only few inspection types but we have added all known types in this, no harm in adding 18 too but it wont make any difference